src/PageContent/Calendar/Header.js

```diff
@@ -1,45 +1,39 @@
 import React, { Component } from 'react';
-import { AccountType, DeliveryType, StringFormat } from '../../Enums';
+import { AccountType, StringFormat } from '../../Enums';
 import './Header.css';
 import white_cross from '../../icons/white_cross.svg';
 import moment from 'moment';
 
 class Header extends Component {
     render() {
-        let deliveryType = '';
-        if (this.props.eventType === DeliveryType.RECURRING) {
-            if (this.props.accountType !== AccountType.RECEIVING_AGENCY) {
-                deliveryType = 'Recurring Pickup';
-            } else {
-                deliveryType = 'Recurring Delivery';
-            }
+        let title = 'Recurring ';
+        if (this.props.accountType !== AccountType.RECEIVING_AGENCY) {
+            title += 'Pickup';
+        } else {
+            title += 'Delivery';
         }
+
         let headerClass = '';
```

Comment:
[just a comment, no need to change]: could use `title +=  this.props.accountType !== AccountType.RECEIVING_AGENCY) ? 'Pickup' : 'Delivery'` and `let headerClass = this.props.futureEvent ? 'header future-header' : 'header past-header'`, which is just a fancy way to save some typing.